triggers/cron.py

```diff
@@ -17,6 +17,7 @@ class CronTrigger():
     """ Notify signals accoriding to cron-like timetable """
 
     cron = ObjectProperty(CronConf, title='Cron Schedule', default=CronConf())
+    utc = BoolProperty(title='UTC', default=False)
```

Comment:
This should default to True.  The block was already using UTC by default, so making it default to not using UTC would cause backward-compatibility issues.